src/pages/home/report/ReportActionsView.js

```diff
@@ -592,6 +605,10 @@ export default compose(
     withLocalize,
     withPersonalDetails(),
     withOnyx({
+        isLoadingReportData: {
+            key: ONYXKEYS.IS_LOADING_REPORT_DATA,
+            initwithStoredValues: false,
```

Comment:
If we are already loading the reports when this component mounts then we won't ever notice that we moved from:

`isLoadingReportData: true` > `isLoadingReportData: false` ?

Reply:
hmm thats a good question, When the component mounts its value willl be true and remain true until we receive data, but I don't think that line will be necessary anymore:
`initwithStoredValues: false,` we can remove it keeping the same behavior.